ingestion_tools/dataset_configs/template.yaml

```diff
@@ -84,6 +84,14 @@ annotations: OPTIONAL
       mask_label: OPTIONAL, INTEGER (DEFAULT 1)
     parent_filters: see InstanceSegmentation.parent_filters
     exclude: SEE InstanceSegmentation.exclude
+  - TriangularMesh:
+      file_format: see InstanceSegmentation.file_format
+      glob_string: see InstanceSegmentation.glob_string
+      glob_strings: see InstanceSegmentation.glob_strings
+      is_visualization_default: see InstanceSegmentation.is_visualization_default
+      scale_factor: OPTIONAL, FLOAT (DEFAULT 1) (POSITIVE)
```

Comment:
Will the `ontology_id` field be added as part of the extension PR?

Reply:
Ontology id was a misnomer. It should be [biological annotation name](https://emdb-empiar.github.io/EMDB-SFF/v0.8.0.dev1/?_=x) which is just a name to describe the mesh as described in the EMDB-SFF spec. It is only needed for the hff file format which can contain multiple meshes per file. So it'll be added in the next PR.

Reply:
I plan to short hand the name to mesh_name in the future PR, that way it can be applied to future mesh formate that may support multiple meshes in a single file.

Should we enforce uniqueness among the `biological_annotation.names` within the hff?

Reply:
We can add uniqueness of the `biological_annotation_names` to the config validation python extension.